src/Tool.tsx

```diff
@@ -1,13 +1,13 @@
-import React from "react";
-import { Icons, IconButton, TooltipMessage, WithTooltip } from "@storybook/components";
-import { PARAM_KEY, PREFIX_PARAM_KEY, ICON_PARAM_KEY, INFO_LINK, TOOL_ID } from "./constants";
-import { useParameter } from '@storybook/manager-api';
-
+import React, { memo } from "react";
+import { RepoIcon } from "@storybook/icons";
+import { useParameter } from "storybook/internal/manager-api";
+import { IconButton, TooltipMessage, WithTooltip } from "storybook/internal/components";
 
+import { PARAM_KEY, PREFIX_PARAM_KEY, ICON_PARAM_KEY, INFO_LINK, TOOL_ID } from "./constants";
 
 const Tooltip = () => (
   <TooltipMessage
-    title="Repository Link"
+    title="Source Link"
```

Comment:
Changed the title of the tooltip to help your users identify the name of the addon responsible for creating this tooltip.